_rules/lists-children-follow-context-model-c6f8a9.md

```diff
@@ -0,0 +1,133 @@
+---
+id: c6f8a9
+name: Child elements of list(s) follow context model
+rule_type: atomic
+description: |
+  This rule checks that child elements of a list follow context model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility tree
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `li`, `dd` and `dt` HTML elements where the [semantic role][] is the same as the [implicit semantic role][].
+
+## Expectation
+
+Each test target element:
+
+- is [owned by][] a node which;
+  - is categorized as [flow content][]; and
+  - has a [semantic role][] from the [required owned element][] list for the test target's [semantic role]().
```

Comment:
This has nothing to do with the HTML content model.
Not a problem per se (still a valid thing to check), but he rule name and description are misleading…

The symmetric relation of the HTML content model is the allowed context (https://html.spec.whatwg.org/multipage/dom.html#concept-element-contexts) In the case of, say, `li` (https://html.spec.whatwg.org/multipage/grouping-content.html#the-li-element) it is indeed `ol`, `ul` and `menu`.

Is the HTML allowed context/content model what you want to check? I am not sure this has a fail-on-fail with 1.3.1 (rather with 4.1.1). Is the ARIA role model what you want to check?